doc/index.html

```diff
@@ -90,7 +90,7 @@
     </div> 
    </div> 
    <div class="row"> 
-    <div class="section-sidebar" style="position: fixed; left: 120px; top: 160px; bottom: 20px;"> 
+    <div class="section-sidebar" style="position: fixed; left: 60px; top: 160px; bottom: 20px;"> 
```

Comment:
This change should be made for all pages in docs and guides, so it should be changed in the page generator probably